xds/internal/client/bootstrap/bootstrap.go

```diff
@@ -136,21 +158,15 @@ type xdsServer struct {
 func NewConfig() (*Config, error) {
 	config := &Config{}
 
-	fName := env.BootstrapFileName
-	if fName == "" {
-		return nil, fmt.Errorf("xds: Environment variable %q not defined", "GRPC_XDS_BOOTSTRAP")
-	}
-	logger.Infof("Got bootstrap file location %q", fName)
-
-	data, err := bootstrapFileReadFunc(fName)
+	data, err := bootstrapConfigFromEnvVariable()
 	if err != nil {
-		return nil, fmt.Errorf("xds: Failed to read bootstrap file %s with error %v", fName, err)
+		return nil, fmt.Errorf("xds: Failed to get bootstrap config, error %v", err)
```

Comment:
Let's try hard to avoid the double-"xds" prefix wherever possible:

```
xds: Failed to get boostrap config (content ____) with error: xds: bootstrap environment variable ("x" or "y") not defined
```

Are we sure we need to print the entire contents of the bootstrap file in the error message, as well?  That seems excessive to me.  We could just log the contents (we do debug log them above already) separately?

Reply:
Removed "xds" prefix from the inner function.

And removed content.